pydebrid/core/alldebrid.py

```diff
@@ -41,7 +41,7 @@ def connect(self, username, password):
         # Current params used for connection
         connect_params = {
             'action': 'login',
-            'returnpage': '/account/',
+            'returnpage': 'account/', # When it's '/account/' the check below assumes it needs captcha. Though it's nothing like that at all.. Just an extra '/'
```

Comment:
Hello, don't you want to submit this fix as a PR for pydebrid?
Thanks in advance :)